core/domain/stats_jobs_one_off.py

```diff
@@ -19,8 +19,10 @@
 import ast
 import collections
 import copy
+import datetime
```

Comment:
General question for this PR: shouldn't there also be validation for the fields in question to ensure they are not None when stored? In general we should protect the datastore from having any bad data written to it at all.

Reply:
@seanlip I just checked. There is validation. But, when no answer is entered, it defaults to "". And when no time is recorded, it defaults to 0. Should we guard against these values? Or better to default them to None instead so that we can catch them during validation?